webapp/src/components/NewsItem.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div>
+    <h1>{{ title }} {{ (votes) }}</h1>
+    <button v-on:click="incrementVotes">Upvote</button>
+    <button v-on:click="decrementVotes">Downvote</button>
+    <button v-on:click="removeMe">Remove</button>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "NewsItem",
+  props:["title","votes"],
+  methods: {
+    incrementVotes() {
+      this.$emit('upvoteItem', this.title)
```

Comment:
This component should be responsible for updating the `votes` on the news item. Not the parent component.